examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -2,16 +2,15 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-
-import {
-    IKeyValueDataObject,
-    KeyValueDataObject,
-} from "@fluid-experimental/data-objects";
-import { Fluid } from "@fluid-experimental/fluid-static";
+import { KeyValueDataObject } from "@fluid-experimental/data-objects";
+import Fluid, { FluidCreateContainerConfig } from "@fluid-experimental/fluid-static";
 import { TinyliciousService } from "@fluid-experimental/get-container";
 import { DiceRollerController } from "./controller";
 import { renderDiceRoller } from "./view";
 
+const service = new TinyliciousService();
+Fluid.init(service);
```

Comment:
Rather than retain state in static members of the class, I think this is just better done with an instance (i.e. everyone uses the `FluidInstance` approach, which we could safely rename to `Fluid` if there's no static/instance distinction to be made).  Which honestly sounds pretty good to me.

Reply:
The static global is an optimization for the 80% scenario where one application will only ever talk to one service.

In this example it's easy to see how you could also use an instance creation pattern. But where I think the global pattern excels is in complicated apps where you will have multiple places where you want to create containers spread out through your application. Having to pass the fluid instance through your application now becomes a task.

The reason I kept FluidInstance was to allow for this scenario if developers want it but if we believe that the mapping in the 80% case is 1:1 with app:service then this pattern makes the developers life easier.